### PR TITLE
Explicit column names in SQL used in `from_table`

### PR DESCRIPTION
**Pull request by KathiaBarahona**
We should just select needed columns from SQL, selecting all columns for BigQuery might be expensive.

### CODE REVIEW

File: bach/bach/dataframe.py [hunk 1]
```diff
@@ -584,7 +584,8 @@ def from_table(
             sql_table_name_template = f'{{bq_project_id}}.{sql_table_name_template}'
             sql_params['bq_project_id'] = quote_identifier(engine.dialect, bq_project_id)
 
-        sql = f'SELECT * FROM {sql_table_name_template}'
```
Comment (jansentom) [inline bach/bach/dataframe.py:587]: The comments on line 575-576 should probably also reflect this change.

**Review by jansentom — APPROVED** [review]
Small comment fix requested, rest LGTM!

File: bach/tests/functional/bach/test_df_from_pandas.py [hunk 2]
```diff
@@ -65,15 +65,16 @@ def test_from_pandas_table(pg_engine):
...
-        engine=pg_engine,
-        df=pdf,
-        convert_objects=True,
-        name='test_from_pd_{table}_"injection"',
-        materialization='table',
-        if_exists='replace'
-    )
-    assert_equals_data(bt, expected_columns=EXPECTED_COLUMNS_INJECTION, expected_data=EXPECTED_DATA_INJECTION)
+
+    with pytest.raises(ValueError, match="unexpected '{' in field name"):
```
Comment (thijs-obj) [inline bach/tests/functional/bach/test_df_from_pandas.py:69]: Question:
1) Why did we change this behaviour?
2) If this is the behaviour we want, should we not test this for both BigQuery and Postgres?

File: bach/bach/dataframe.py (hunk 1 above)
Comment (thijs-obj) [reply bach/bach/dataframe.py:587]: agree

**Review by thijs-obj — COMMENTED** [review]
Added a comment

File: bach/tests/functional/bach/test_df_from_pandas.py (hunk 2 above)
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_df_from_pandas.py:69]: doesn't make sense to me to allow it if user is not able to select the series with those type of names :woman_shrugging:  It is actually tested for both

File: bach/bach/dataframe.py (hunk 1 above)
Comment (KathiaBarahona) [reply bach/bach/dataframe.py:587]: when did those comments appeared haha Will change them

**Review by thijs-obj — COMMENTED** [review]
lgtm

**Pull request merged**